apps/api/src/app/shared/services/logger.service.ts

```diff
@@ -1,14 +1,20 @@
 import { createNestLoggingModuleOptions, PinoLogger } from '@novu/application-generic';
 import packageJson from '../../../../package.json';
 
-export const getLogger = (context: string) => {
-  const logger = new PinoLogger(
-    createNestLoggingModuleOptions({
-      serviceName: packageJson.name,
-      version: packageJson.version,
-    })
-  );
-  logger.setContext(context);
-
-  return logger;
+let loggerInstance: PinoLogger | null = null;
+
+export const getLogger = () => {
+  if (!loggerInstance) {
+    loggerInstance = new PinoLogger(
+      createNestLoggingModuleOptions({
+        serviceName: packageJson.name,
+        version: packageJson.version,
+        silent: true,
```

Comment:
we’re making this init silent to avoid second log of the configuration, first LoggerModule initialization. the second init happens here manually, and that’s fine, because we know only one additional instance will be created.
we’re also reusing createNestLoggingModuleOptions, so the same config will be generated consistently.